core/server/src/main/java/alluxio/master/block/BlockMasterClientRestServiceHandler.java

```diff
@@ -43,10 +48,20 @@
   public static final String SERVICE_VERSION = "service_version";
   public static final String GET_BLOCK_INFO = "block_info";
   public static final String GET_CAPACITY_BYTES = "capacity_bytes";
+  public static final String GET_DISK_CAPACITY_BYTES = "disk_capacity_bytes";
```

Comment:
why are these endpoints added under the master/block namespace? I think they should all go directly under the master namespace ... they have no relation to the block master